Dockerfile.assisted-service-build

```diff
@@ -1,7 +1,12 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.16
+FROM quay.io/edge-infrastructure/postgresql-12-centos7:latest
 
-ENV GO111MODULE=on
-ENV GOFLAGS=""
```

Comment:
Do we still need to set these?

Reply:
no, those are leftovers. I will remove it.

Reply:
What is the other scenario that might need it? Maybe we can add later if it breaks something but as of now I don't know any other scenario than unit tests and it doesn't need it.

Reply:
:man_shrugging: It looks like there was an existing `GOFLAG` value in the previous base image that we were probably trying to remove.

As for `GO111MODULE` it looks like it's on by default as of go 1.16 ref: https://go.dev/blog/go116-module-changes